examples/quickstart.php

```diff
@@ -55,28 +52,22 @@ public static function nameNew($username)
             return $instance;
         }
 
-        /**
-         * @return Uuid
-         */
-        public function userId()
+        public function userId(): Uuid
         {
             return $this->uuid;
         }
 
-        /**
-         * @return string
-         */
-        public function name()
+        public function name(): string
         {
             return $this->name;
         }
 
         /**
          * @param $newName
          */
-        public function changeName($newName)
+        public function changeName(string $newName): void
         {
-            \Assert\that($newName)->notEmpty()->string();
+            Assertion::notEmpty($newName);
```

Comment:
This line should be replaced, since the "beberlei/assert" dependency is not available in composer anymore.